react-1/src/components/User.js

```diff
@@ -0,0 +1,119 @@
+import React, {useImperativeHandle, useRef} from 'react';
+import { DragSource, DropTarget } from 'react-dnd';
+import {createUseStyles} from 'react-jss';
+import { useSpring, animated } from 'react-spring';
```

Comment:
Did you check if you also could have used `useSprings` to have something like https://codesandbox.io/embed/r5qmj8m6lq ?

Reply:
this example does with static list item data, i'd tried to implement that codepen but, didn't acheived it due to that